src/components/AddItem.js

```diff
@@ -24,25 +24,26 @@ const AddItem = () => {
     e.preventDefault();
     const alreadyExists = await existingItemCheck(groceryItem);
     if (alreadyExists) {
-      alert('Item already on shoppping list');
+      alert("Item already on shoppping list");
       return;
     }
-    db.collection(localStorage.getItem('token')).add({
+    db.collection(localStorage.getItem("token")).add({
       itemName: groceryItem,
       daysToPurchase: daysToPurchase,
       lastPurchasedDate: null,
+      numberOfPurchases: 0,
```

Comment:
It looks like we are still having issues with competing linters for the quotes thing. This is the only line that changed in this file - adding a 'numberOfPurchases' key to the db for each item.

Reply:
Maybe we need to manually change all our .prettierrc files on our next dev branch?